internal/ignition/installmanifests.go

```diff
@@ -182,6 +180,7 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte)
 	//cleanup resources at the end
 	defer release.Cleanup(ctx)
 
+	// Critical section -- start
```

Comment:
The "critical section" of concurrent code is the part that cannot be run in multiple threads at once. That's not the case for this so I'm not sure what you're trying to indicate with these comments.

Reply:
Apologies, this is just something that should not have ended up in the PR as I refactor to respond to comments.
Please disregard it for now. Was not meant to be here.